### DISCUSSION TITLE
Who can help with the wiring 🆘

### DISCUSSION THREAD

**Discussion by felix-exon** [tbnobody/OpenDTU#668] [Q&A]
![image](https://user-images.githubusercontent.com/24978665/223557522-bc77e763-6cef-4390-b110-c1f518a9fac0.png)
![image](https://user-images.githubusercontent.com/24978665/223557533-78099a22-599e-4cda-8b73-33133821d7ba.png)

My board seems to be different than the one described in the documentation. Can any1 help me with the wiring (sorry, total noob here).

Any help highly appreciated!

**Comment by Harald-U** [comment]
Look at the schematics in the main README:

 ![schematics](https://github.com/tbnobody/OpenDTU/raw/master/docs/Wiring_ESP32_Schematic.png) 

For example, the CE pin (pin number 3)  of the NRF24L01+ is connected to GPIO4 of the ESP32. On your ESP32, GPIO4 is labelled D4 which is the 5th pin from bottom on the right side. 

Equally, GPIO23 is D23 on your board and so on.

HTH, Harald
